windows/snowagent.config

```diff
@@ -120,5 +122,7 @@
         <!-- <Setting key="env.java_home" value="java\bin\java.exe" /> -->
         <!-- <Setting key="env.is_virtual_desktop_infrastructure" value="false" /> -->
         <!-- <Setting key="disable_all_updates" value="false" /> -->
+		<!-- Control process affinity to limit CPU usage during scan-->
+		<!-- <Setting key="process_affinity" value="" /> -->
```

Comment:
If we add this here, could we maybe use a valid value?

Reply:
empty is the default value, as per documentation
https://docs.snowsoftware.io/other-snow-products/snow-inventory-agents-and-oracle-scanners/agent-configuration-file/configuration-reference#systemsettings

Reply:
@gpcarval so why do we need to add an commented line with default value here? The config file should not just be the repetition of default values.